modules/standard/Map.chpl

```diff
@@ -875,7 +998,15 @@ module Map {
     :rtype: `bool`
   */
   operator map.!=(const ref a: map(?kt, ?vt, ?ps),
-                  const ref b: map(kt, vt, ps)): bool {
+                  const ref b: map(kt, vt, ps)): bool throws
+    where mapMethodsThrow {
+    return !(a == b);
```

Comment:
Since this relies on `==`, if that doesn't actually need to throw then this one doesn't either.